### PR TITLE
word concatenation test

### PR DESCRIPTION
**Pull request by lidoror**

### CODE REVIEW

File: python_katas/kata_1/test.py [hunk 1]
```diff
@@ -168,5 +180,5 @@ def test_sample(self):
 if __name__ == '__main__':
     import inspect
     import sys
-    unittest_runner(inspect.getmembers(sys.modules[__name__], inspect.isclass))
 
+    unittest_runner(inspect.getmembers(sys.modules[__name__], inspect.isclass))
```
Comment (alonitac) [inline python_katas/kata_1/test.py:184]: Redundant change

File: python_katas/kata_1/test.py [hunk 2]
```diff
@@ -30,9 +30,21 @@ class TestWordsConcatenation(unittest.TestCase):
     1 Katas
     """
 
-    def test_sample(self):
-        # your code here
-        pass
+    def test_empty_list(self):
+        lst = []
+        self.assertEqual(questions.words_concatenation(lst), [])
```
[the remark this answers is not in the document]
Comment (lidoror) [reply python_katas/kata_1/test.py]: i fixed it

File: python_katas/kata_1/test.py (hunk 1 above)
Comment (lidoror) [reply python_katas/kata_1/test.py]: i didnt notice that change fixed it

**Comment by lidoror** [comment]
changes fixed

**Review by alonitac — APPROVED** [review]

**Pull request merged**